pkgs/by-name/ch/chirpstack-fuota-server/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  name = "chirpstack-fuota-server";
```

Comment:
There is no stable release yet for this component of the Chirpstack stack.

I've created a ticket upstream to release a new version in https://github.com/chirpstack/chirpstack-fuota-server/issues/17

Reply:
You should use the current standard for unstable versions. Since there isn't any tagged release that would be `0-unstable-YYYY-MM-DD`. With the author date of the latest commit you're packaging.

See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
Thanks. I've addressed it in the latest version of the MR